src/topp/CometAdapter.cpp

```diff
@@ -633,11 +634,9 @@ class TOPPCometAdapter :
     }
 
     // determine type of spectral data (profile or centroided)
-    SpectrumSettings::SpectrumType spectrum_type = exp[0].getType();
-
-    if (spectrum_type == SpectrumSettings::PROFILE)
+    for (const auto& s : exp)
     {
-      if (!getFlag_("force"))
+      if (s.getType() == SpectrumSettings::PROFILE && !getFlag_("force"))
```

Comment:
@jpfeuffer changed by me

Reply:
weird. gitpod did not store my last save correctly I guess.

Reply:
thx